project/make/fedora

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+DEST=$1
+
+# source the .bundle file
+source "$(dirname "$BASH_SOURCE")/.bundle"
+
+# Build docker as a fedora package using FPM and REPREPRO (sue me).
+# bundle_binary must be called first.
+bundle_fedora() {
+	DIR=$DEST/build
+
+	# include udev rules, contributed init, completions & man pages
+	bundle_udev
+	bundle_init
+	bundle_completions
+	bundle_manpages
+
+	# Copy the binary
+	# This will fail if the binary bundle hasn't been built
+	mkdir -p $DIR/usr/bin
+	cp $DEST/../binary/docker-$VERSION $DIR/usr/bin/docker
+
+	# Generate postinst/prerm/postrm scripts
+	cat > $DEST/postinst <<'EOF'
+#!/bin/sh
+set -e
+set -u
+
+if [ "$1" = 'configure' ] && [ -z "$2" ]; then
+	if ! getent group docker > /dev/null; then
+		groupadd --system docker
+	fi
+fi
+
+if ! { [ -x /sbin/initctl ] && /sbin/initctl version 2>/dev/null | grep -q upstart; }; then
+	# we only need to do this if upstart isn't in charge
+	update-rc.d docker defaults > /dev/null || true
+fi
+if [ -n "$2" ]; then
+	_dh_action=restart
+else
+	_dh_action=start
+fi
+service docker $_dh_action 2>/dev/null || true
+
+#DEBHELPER#
+EOF
+	cat > $DEST/prerm <<'EOF'
+#!/bin/sh
+set -e
+set -u
+
+service docker stop 2>/dev/null || true
+
+#DEBHELPER#
+EOF
+	cat > $DEST/postrm <<'EOF'
+#!/bin/sh
+set -e
+set -u
+
+if [ "$1" = "purge" ] ; then
+	update-rc.d docker remove > /dev/null || true
+fi
+
+# In case this system is running systemd, we make systemd reload the unit files
+# to pick up changes.
+if [ -d /run/systemd/system ] ; then
+	systemctl --system daemon-reload > /dev/null || true
+fi
+
+#DEBHELPER#
+EOF
+	# TODO swaths of these were borrowed from debhelper's auto-inserted stuff, because we're still using fpm - we need to use debhelper instead, and somehow reconcile Ubuntu that way
+	chmod +x $DEST/postinst $DEST/prerm $DEST/postrm
+
+	(
+		# switch directories so we create *.rpm in the right folder
+		cd $DEST
+
+		# create lxc-docker-VERSION package
+		fpm -s dir -C $DIR \
+			--name lxc-docker-$VERSION --version "$PKGVERSION" \
+			--after-install $DEST/postinst \
+			--before-remove $DEST/prerm \
+			--after-remove $DEST/postrm \
+			--architecture "$PACKAGE_ARCHITECTURE" \
+			--prefix / \
+			--depends iptables \
+			--description "$PACKAGE_DESCRIPTION" \
+			--maintainer "$PACKAGE_MAINTAINER" \
+			--conflicts docker \
+			--conflicts docker.io \
```

Comment:
This particular line definitely isn't necessary, but having `docker-io` here might be.